src/main/java/edu/nextstep/calculator/domain/Numbers.java

```diff
@@ -0,0 +1,20 @@
+package edu.nextstep.calculator.domain;
+
+public class Numbers {
+	private final Number[] numbers;
+
+	public Numbers(String[] textSplitResult) {
+		numbers = new Number[textSplitResult.length];
+		for (int i = 0; i < textSplitResult.length; i++) {
+			numbers[i] = Number.createFromString(textSplitResult[i]);
+		}
+	}
+
+	public int sumAll() {
+		Number result = Number.createFromInteger(0);
+		for (Number number : numbers) {
+			result = result.sum(number);
+		}
```

Comment:
아래와 같이 Stream을 활용해볼 수 있을 것 같아요~

```suggestion
		Number result = Arrays.stream(numbers)
				.reduce(Number.createFromInteger(0), Number::sum);
```

Reply:
reduce 추천 감사드립니다! stream이 정말 가독성 측면도 훨씬 좋네요!